disperser/cmd/controller/main.go

```
@@ -217,5 +230,22 @@ func RunController(ctx *cli.Context) error {
 		}
 	}()
 
+	// Create readiness probe file once the controller starts successfully
+	if _, err := os.Create(config.ControllerReadinessProbePath); err != nil {
+		logger.Warn("Failed to create readiness file", "error", err, "path", config.ControllerReadinessProbePath)
+	}
+
+	if _, err := os.Create(config.ControllerHealthProbePath); err != nil {
+		logger.Warn("Failed to create healthProbe file: %v", err)
+	}
+
+	// Start heartbeat monitor
+	go func() {
+		err := healthcheck.HeartbeatMonitor(config.ControllerHealthProbePath, controllerMaxStallDuration, controllerLivenessChan, logger)
+		if err != nil {
+			logger.Warn("Failed to start heartbeatMonitor: %v", err)
```

I think this case definitely deserves to fail more loudly (Error level)
If HeartbeatMonitor does not run but component keeps sending heartbeat, the component can eventually stall because the channel can be full. Is my understanding correct? 
We should fail loudly here and maybe close the channel as well